mne/viz/evoked.py

```diff
@@ -1191,3 +1191,342 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     # show and return it
     plt_show(show)
     return fig
+
+
+def _ci(arr, ci):
+    """Calculate the `ci`% parametric confidence interval for `arr`"""
+    from scipy import stats
+    mean, sigma = arr.mean(0), stats.sem(arr, 0)
+    # This is highly convoluted to support 17th century Scipy
+    # XXX Fix when Python 2.6 support is dropped!
+    return np.asarray([stats.t.interval(ci, arr.shape[0],
+                       loc=mean_, scale=sigma_)
+                       for mean_, sigma_ in zip(mean, sigma)]).T
+
+
+def plot_compare_evokeds(evokeds, picks=None, conditions=None, ch_names=None,
+                         colors=None, linestyles=['-'], styles=None,
+                         vlines=[0], ci=0.95, truncate_yaxis=True, ymin=None,
+                         ymax=None, invert_y=False, ax=None, title=None,
+                         show=True):
+    """ Plot, for multiple conditions, individual evoked time courses of one or
+    the mean of multiple sensors
+
+    This function is useful for comparing ER[P/F]s at a specific location. It
+    plots grand averages or, if supplied with a list/dict of lists of evoked
+    instances, grand averages plus confidence intervals.
+
+    Parameters
+    ----------
+    evokeds : instance of mne.Evoked | list | dict
+        If a single evoked instance, it is plotted as a time series.
+        If a dict, the contents are plotted as single time series each and the
+        keys are used as condition labels.
+        If a [dict/list] of lists, the unweighted mean is plotted as a time
+        series and the parametric confidence interval is plotted as a shaded
+        area. All instances must have the same shape - channel numbers, time
+        points etc.
+    picks : int | list of int | str | list of str | None
+        The indices (int) or names (str) of the sensors to plot. Must all
+        have the same channel type.
+        If the selected channels are gradiometers, the corresponding pairs
+        will be selected.
+        If None, the global field power is plotted instead.
+    conditions : list of str | None
+        If evokeds is a single instance or a list (but not a dictionary),
+        this can be used for legend labels.
+    colors : list | dict | None
+        If a list, will be sequentially used for line colors.
+        If a dict, can map evoked keys or HED-like tags to conditions. For
+        example, if `evokeds` is a dict with the keys "Aud/L", "Aud/R",
+        "Vis/L", "Vis/R", `colors` can be dict(Aud='r', Vis='b') to map both
+        Aud/L and Aud/R to the color red.
+        If None, a sequence of desaturated colors is used.
```

Comment:
"Default is None" or something like this. More generally - I'm not sure how often the defaults are mentioned in the mne docstrings, but it might be good to do so.